oh_staff_ui/models.py

```diff
@@ -258,3 +258,32 @@ class ItemPublisherUsage(models.Model):
     type = models.ForeignKey(
         PublisherType, on_delete=models.PROTECT, blank=False, null=False
     )
+
+
+class CopyrightType(models.Model):
+    type = models.CharField(max_length=256, blank=False, null=False)
+
+    def __str__(self):
+        return self.type
+
+
+class Copyright(models.Model):
+    value = models.CharField(max_length=256, blank=False, null=False)
+    source = models.ForeignKey(
+        AuthoritySource, on_delete=models.PROTECT, blank=False, null=False
+    )
+
+    def __str__(self):
+        return self.value
+
+
+class ItemCopyrightUsage(models.Model):
+    item = models.ForeignKey(
+        ProjectItem, on_delete=models.PROTECT, blank=False, null=False
+    )
+    publisher = models.ForeignKey(
```

Comment:
@kjallen Didn't notice this before, sorry, but `publisher` here seems wrong.  Should this be `copyright` or `copyright_statement` or something else?

Reply:
Thank you for catching @akohler, corrected and updated.